--- node_modules/react-dom/cjs/react-dom-server.node.development.js
+++ node_modules/react-dom/cjs/react-dom-server.node.development.js
@@ -1133,7 +1133,7 @@
   cellpadding: 'cellPadding',
   cellspacing: 'cellSpacing',
   challenge: 'challenge',
-  charset: 'charSet',
+  charset: 'charset',
   checked: 'checked',
   children: 'children',
   cite: 'cite',
@@ -1244,7 +1244,7 @@
   src: 'src',
   srcdoc: 'srcDoc',
   srclang: 'srcLang',
-  srcset: 'srcSet',
+  srcset: 'srcset',
   start: 'start',
   step: 'step',
   style: 'style',
