--- node_modules/gatsby-plugin-sharp/index.js
+++ node_modules/gatsby-plugin-sharp/index.js
@@ -12,6 +12,25 @@
 
 var _asyncToGenerator3 = _interopRequireDefault(_asyncToGenerator2);
 
+function mkdirp (p, made) {
+  var mode = parseInt('0777', 8) & (~process.umask());
+  if (!made) made = null;
+  p = path.resolve(p);
+  try {
+    fs.mkdirSync(p, mode);
+    made = made || p;
+  }
+  catch (err0) {
+    switch (err0.code) {
+      case 'ENOENT' :
+        made = mkdirp(path.dirname(p), made);
+        mkdirp(p, made);
+        break;
+    }
+  }
+  return made;
+};
+
 var notMemoizedbase64 = function () {
   var _ref4 = (0, _asyncToGenerator3.default)( /*#__PURE__*/_regenerator2.default.mark(function _callee2(_ref3) {
     var file = _ref3.file,
@@ -270,7 +289,7 @@
             }).src;
             srcSet = images.map(function (image) {
               return `${image.src} ${Math.round(image.width)}w`;
-            }).join(`,\n`);
+            }).join(`,`);
             originalName = file.base;
             return _context4.abrupt("return", {
               base64: base64Image.src,
@@ -327,9 +346,7 @@
             sizes = [];
 
             sizes.push(options.width);
-            sizes.push(options.width * 1.5);
             sizes.push(options.width * 2);
-            sizes.push(options.width * 3);
             dimensions = getImageSize(file);
             filteredSizes = sizes.filter(function (size) {
               return size <= dimensions.width;
@@ -395,7 +412,7 @@
                 default:
               }
               return `${image.src} ${resolution}`;
-            }).join(`,\n`);
+            }).join(`,`);
             originalName = file.base;
             return _context5.abrupt("return", {
               base64: base64Image.src,
@@ -855,8 +872,9 @@
 
   var argsDigestShort = argsDigest.substr(argsDigest.length - 5);
 
-  var imgSrc = `/${file.name}-${file.internal.contentDigest}-${argsDigestShort}.${fileExtension}`;
-  var filePath = path.join(process.cwd(), `public`, `static`, imgSrc);
+  var imgSrc = `/${file.internal.contentDigest.slice(0,5)}-${argsDigestShort}.${fileExtension}`;
+  mkdirp(path.join(process.cwd(), `public`, `img`, `${file.relativeDirectory}`, `${file.name}`))
+  var filePath = path.join(process.cwd(), `public`, `img`, `${file.relativeDirectory}`, `${file.name}`, imgSrc);
 
   // Create function to call when the image is finished.
   var outsideResolve = void 0,
@@ -901,7 +919,7 @@
   queueJob(job, reporter);
 
   // Prefix the image src.
-  var prefixedSrc = options.pathPrefix + `/static` + imgSrc;
+  var prefixedSrc = options.pathPrefix + `/img/` + file.relativeDirectory + `/` + file.name + imgSrc;
 
   return {
     src: prefixedSrc,
