--- node_modules/gatsby/cache-dir/static-entry.js
+++ node_modules/gatsby/cache-dir/static-entry.js
@@ -263,6 +263,7 @@
   // Taken from https://www.html5rocks.com/en/tutorials/speed/script-loading/
   const scriptsString = scripts
     .map(s => `"${pathPrefix}${JSON.stringify(s).slice(1, -1)}"`)
+    .map(s => s.replace(/\/+/g, "/"))
     .join(`,`)
   postBodyComponents.push(
     <script
--- node_modules/gatsby/dist/redux/reducers/webpack.js
+++ node_modules/gatsby/dist/redux/reducers/webpack.js
@@ -8,7 +8,6 @@
       {
         let nextConfig = action.payload;
         delete nextConfig.entry;
-        delete nextConfig.output;
         delete nextConfig.target;
         delete nextConfig.resolveLoaders;
         return merge(state, nextConfig);
