--- node_modules/gatsby/cache-dir/app.js
+++ node_modules/gatsby/cache-dir/app.js
@@ -34,7 +34,7 @@
     })
   }
 
-  const rootElement = document.getElementById(`___gatsby`)
+  const rootElement = document.getElementById(`root`)
 
   let Root = require(`./root`)
   if (Root.default) {
--- node_modules/gatsby/cache-dir/default-html.js
+++ node_modules/gatsby/cache-dir/default-html.js
@@ -17,7 +17,7 @@
           {this.props.preBodyComponents}
           <div
             key={`body`}
-            id="___gatsby"
+            id="root"
             dangerouslySetInnerHTML={{ __html: this.props.body }}
           />
           {this.props.postBodyComponents}
--- node_modules/gatsby/cache-dir/production-app.js
+++ node_modules/gatsby/cache-dir/production-app.js
@@ -191,7 +191,7 @@
       ReactDOM.hydrate(
         <NewRoot />,
         typeof window !== `undefined`
-          ? document.getElementById(`___gatsby`)
+          ? document.getElementById(`root`)
           : void 0,
         () => {
           apiRunner(`onInitialClientRender`)
--- node_modules/history/PathUtils.js
+++ node_modules/history/PathUtils.js
@@ -2,6 +2,7 @@
 
 exports.__esModule = true;
 var addLeadingSlash = exports.addLeadingSlash = function addLeadingSlash(path) {
+  path = path.replace(/^\/+/, "/")
   return path.charAt(0) === '/' ? path : '/' + path;
 };
 
