--- node_modules/gatsby/dist/bootstrap/load-plugins/load.js
+++ node_modules/gatsby/dist/bootstrap/load-plugins/load.js
@@ -133,7 +133,7 @@
     }; // Add internal plugins
 
 
-    const internalPlugins = [`../../internal-plugins/dev-404-page`, `../../internal-plugins/load-babel-config`, `../../internal-plugins/component-page-creator`, `../../internal-plugins/internal-data-bridge`, `../../internal-plugins/prod-404`, `../../internal-plugins/query-runner`];
+    const internalPlugins = [`../../internal-plugins/dev-404-page`, `../../internal-plugins/load-babel-config`, `../../internal-plugins/internal-data-bridge`, `../../internal-plugins/prod-404`, `../../internal-plugins/query-runner`];
     internalPlugins.forEach(relPath => {
       const absPath = path.join(__dirname, relPath);
       plugins.push(processPlugin(absPath));
@@ -155,6 +155,26 @@
         plugins: []
       }
     });
+
+    const pluginCreatePages = {
+      "path": slash (path.dirname (require.resolve ("@ptb/create-pages"))),
+      "version": require (path.join (
+        path.dirname (require.resolve ("@ptb/create-pages")),
+        "package.json"
+      )).version
+    }
+    
+    plugins.push ({
+      "id": "Plugin @ptb/create-pages",
+      "name": "@ptb/create-pages",
+      "pluginOptions": {
+        "path": "src/pages/index.js",
+        "plugins": []
+      },
+      "resolve": pluginCreatePages.path,
+      "version": pluginCreatePages.version
+    })
+
     return plugins;
   });
 
